client/my-sites/plans-features-main/index.jsx

```diff
@@ -84,6 +90,27 @@ export class PlansFeaturesMain extends Component {
 		}
 	}
 
+	isJetpackBackupAvailable() {
+		const { displayJetpackPlans, jetpackSupportsBackupProducts, siteId } = this.props;
+
+		// Jetpack Backup products are currently under a feature flag
+		if ( ! isEnabled( 'plans/jetpack-backup' ) ) {
+			return false;
+		}
+
+		// Only for Jetpack, non-atomic sites
+		if ( ! displayJetpackPlans ) {
+			return false;
+		}
+
+		// Only for sites with Jetpack >= 7.9alpha
+		if ( siteId && ! jetpackSupportsBackupProducts ) {
```

Comment:
Why do we need `siteId` here?

Reply:
Because we still want to display the backup products UI on the `/jetpack/connect/store` page, and that doesn't have a selected site ID.

Reply:
Interesting but do we know what version the site has at that point already?

Reply:
That's the thing - we don't have a site on that page. So we should be good to display it regardless.